client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -153,6 +173,20 @@ public void builder_authenticator() {
                 BasicAuthHelper.createHeader("test-client", "b3f202cb-29fe-4d13-afb8-15e787c6e56c"));
     }
 
+    @Test
+    public void builder_authenticatorFromString() {
+        TestSupplier supplier = new TestSupplier();
```

Comment:
nit: maybe tiny opportunity to move the common code used in this test and the one before it into a method that gets used by both.    But I realize that comment might apply to other tests as well and unnecessarily increase the scope of this PR.

I think maybe just this test looked bigger in its duplication because of the number of lines.